### PR TITLE
Update OpenRefine logo

### LINKED ISSUE #1453: Update OpenRefine logo
- [ ] We should add the new logo to the UI
- [ ] And also to our README.md
- [ ] And finally our Website

### PR DESCRIPTION
**Pull request by joanneong**
Fixes #1453 

- [x] Add new logo to OpenRefine UI
- [x] Add new logo to `README.md`
- [ ] Add new logo to website

I've updated the OpenRefine logo as well as the favicon displayed on the browser tab. Here is a visual comparison of the changes made (left image is pre-changes, right image is post-changes):

**Index page**
index_page

**Project page**
project

**Preferences page**
preferences

**About page**
about

**Favicon**
favicon

It seems that the new logo is smaller than the old one, but I did not enlarge it because I wanted to keep the height of the logo consistent, and this height is constrained by the division in the index page. 

Also, the logo I used for the replacement in the index page was edited by me; the original had white fonts so I changed it to black via an image editor. Alternatively, we can retain the original slogan like so:

index_top_logo1

instead of what I did:

index_top_logo2

Please let me know your thoughts, and I'd be happy to delete the old logo images if we do not want to keep them around.

### CODE REVIEW

**Comment by wetneb** [comment]
Very nice! But… Am I the only one to think that the font used for the logo looks quite bad in this context? (It's not about your work Joanne obviously). I would say the font is too small and too bold.

**Comment by ettorerizza** [comment]
Mmh, yes, this font is a little cheap I think. It looks a bit "Times New Roman bold and the deal is done". I'm sure we can find a better one by looking a little.

**Comment by thadguidry** [comment]
The artist who designed it gave the reasoning behind the choosing of the "Swift" font with serifs (embellishments), so that artistic variations could be done on the serifs, etc. with patterns, motifs, etc.

[The PDF design doc](https://drive.google.com/file/d/1BrYfAeezccYx2ud4cx5OUbGj8VPQGH2M/view)  that was supplied with it shows the usage of the serifs for embellishment.

I would just enlarge the font for now.  Joanne you should have the original vector files in that ZIP file I emailed you, to work with to increase the font size a bit.

**Comment by joanneong** [comment]
I've tried to edit the logo a little by enlarging the "OpenRefine" in the logo. After making that change, the various pages now look like this:

**Index page**
index_edit2

**Project page**
project_edit2

**Preferences page**
preferences_edit2

**About page**
about_edit2

For the index page I have decided to revert to the original text slogan because I realised that the slogan can actually be translated into a different language. I also left the favicon and the second diamond logo in the index page untouched from when I first opened my pull request yesterday.

@wetneb @ettorerizza @thadguidry What do you make of this?

**Comment by thadguidry** [comment]
@joanneong I think its the perfect font sizing now for our current UI on those pages. Good Job !

**Comment by jackyq2015** [comment]
@joanneong Nice job! Just one minor thing you can consider for the image file names. Instead of logo-openrefine-new.png, may be something like logo-openrefine-40.png to use the previous naming convention. Just an example.

**Comment by thadguidry** [comment]
@joanneong You can delete them Joanne, we have them backed up in Git and elsewhere.

**Comment by joanneong** [comment]
Also, I have updated the logo for `README.md`, but the link is absolute so I can't see it until it's merged into master. I am trying to see if I can change it into a relative link on the markdown page but somehow it's not working yet...

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ OpenRefine is a power tool that allows you to load data, understand it,
 clean it up, reconcile it, and augment it with data coming from
 the web. All with the comfort and privacy of your own computer.
 
-[<img src="https://github.com/OpenRefine/OpenRefine/blob/master/graphics/icon/open-refine-256px.png" align="right">](http://openrefine.org)
+[<img src="./graphics/icon/open-refine-256px.png" style="width: 320px;" align="right">](http://openrefine.org)
```
Comment (thadguidry) [inline README.md]: I think we need tokeep the full expanded URL, right ?  That way a local MD file reader and Github views will work just fine.

Comment (joanneong) [reply README.md]: Oh I see, once I verify that the logo is working I will switch it back to an absolute link then :)

**Comment by joanneong** [comment]
Here's the `README.md` before the update:

readme_original

...and the `README.md` after the update:

readme_new

The new logo is narrower but longer so I didn't try to enlarge it that much. Let me know if this is fine :)

**Comment by thadguidry** [comment]
@joanneong Yes, its great ! Good job Joanne !  Are you ready for final merge ? or do you have more work to do and for us to review on this pull request (PR) ?

**Comment by joanneong** [comment]
@thadguidry nope, I believe this PR is ready for review and merging. Meanwhile, I will be opening another pull request for task 3 (Add new logo to website) [here](https://github.com/OpenRefine/openrefine.github.com) instead.

**Pull request merged**